tests/foreman/ui/test_contentview.py

```diff
@@ -847,11 +846,45 @@ def test_negative_add_dupe_modules(self):
 
         @assert: User cannot add modules multiple times to the view
 
-        @caseautomation: notautomated
-
-
         @CaseLevel: Integration
         """
+        cv_name = gen_string('alpha')
+        module_name = 'samba'
+        product = entities.Product(organization=self.organization).create()
+        puppet_repository = entities.Repository(
+            url=FAKE_0_PUPPET_REPO,
+            content_type='puppet',
+            product=product
+        ).create()
+        puppet_repository.sync()
+        with Session(self.browser) as session:
+            # Create content-view
+            make_contentview(session, org=self.organization.name, name=cv_name)
+            self.assertIsNotNone(self.content_views.search(cv_name))
+            self.content_views.add_puppet_module(
+                cv_name,
+                module_name,
+                filter_term='Latest'
+            )
+            # ensure that the puppet module is added to content view
+            session.nav.go_to_select_org(self.organization.name)
```

Comment:
this not the fault of the test in question and perhaps, the base function add_puppet_module should be reviewed carefully, so delete it